config/convert2rhel.ini

```diff
@@ -19,7 +19,6 @@
 [inhibitor_overrides]
 # incomplete_rollback              = false
 # tainted_kernel_module_check_skip = false
-# outdated_package_check_skip      = false
```

Comment:
Probably I would also remove the `outdated_package_check_skip` from the `/toolopts/config.py` (mapping and `FileConfig`). And depending what get merged first also from [`/utils/__init__.py` (`warn_deprecated_env()`)](https://github.com/oamg/convert2rhel/pull/1387/files#diff-a180a3ef19514b4d0fe0a42f5d5e60abdfdce708992bcd2363de752aeaf30561R1128)